src/main.rs

```diff
@@ -99,8 +99,9 @@ impl OutWriter {
 impl OutWriter {
     pub async fn println(&self, content: impl AsRef<[u8]>) {
         let stderr = &mut *self.stderr.lock().await;
-        stderr.write_all(content.as_ref()).await.unwrap();
-        stderr.write_all(b"\n").await.unwrap();
+        let mut buf: Vec<u8> = content.as_ref().to_vec();
```

Comment:
much allocations :(

Reply:
Yeah, agreed.  But this isn't very critical.  We don't output much.

Reply:
Actually, I think you could the signature to take a `String` and just extend that.

Reply:
that's nicer indeed.  done